draft-ietf-quic-v2.md

```diff
@@ -178,17 +178,31 @@ server MUST include the relevant transport parameters to validate that the
 server sent the Retry and the connection IDs used in the exchange, as described
 in {{Section 7.3 of QUIC}}.
 
-The server SHOULD start sending its Initial packets using the negotiated
-version as soon as it decides to change. Before the server is able to process
-transport parameters from the client, it might need to respond to Initial
-packets from the client. For these packets the server uses the original version.
-
-Once the client has processed a packet using the negotiated version, it SHOULD
-send subsequent Initial packets using that version. The server MUST NOT discard
-its original version Initial receive keys until it successfully processes a
+The server cannot send CRYPTO frames until it has processed the client's
+transport parameters. Therefore, the server MUST send all CRYPTO frames using
```

Comment:
This "Therefore" has the same problem I was describing with the "As" in the earlier text - the second sentence here doesn't derive from the first.
```suggestion
transport parameters. The server MUST send all CRYPTO frames using
```